dbt_project.yml

```diff
@@ -19,8 +19,9 @@ vars:
     ar_invoice_item:              "{{ source('sage_intacct','ar_invoice_item') }}"
     ar_invoice:                   "{{ source('sage_intacct','ar_invoice') }}"
     
-    account_pass_through_columns: []
-    
-clean-targets:
-    - target
-    - dbt_modules
+    account_pass_through_columns: [] ## Reminder to update the name
```

Comment:
Hi @fivetran-joemarkiewicz , could you clarify what you meant by updating the name here; did you mean changing the config name itself to something like "gl_account_pass_through_columns"?